curvefs/src/client/curve_fuse_op.cpp

```diff
@@ -312,7 +312,7 @@ void FuseOpRead(fuse_req_t req, fuse_ino_t ino, size_t size, off_t off,
         return;
     }
 
-    struct fuse_bufvec bufvec;
+    struct fuse_bufvec bufvec = FUSE_BUFVEC_INIT(size);
```

Comment:
```
struct fuse_bufvec bufvec = FUSE_BUFVEC_INIT(rSize);
bufvec.buf[0].mem = buffer.get();
```
may be better

Reply:
```
    bufvec.count = 1;
    bufvec.off = 0;
    bufvec.idx = 0;
    bufvec.buf[0].size = rSize;
```
is unnecessary

Reply:
```
/* Initialize bufvec with a single buffer of given size */
#define FUSE_BUFVEC_INIT(size__)				\
	((struct fuse_bufvec) {					\
		/* .count= */ 1,				\
		/* .idx =  */ 0,				\
		/* .off =  */ 0,				\
		/* .buf =  */ { /* [0] = */ {			\
			/* .size =  */ (size__),		\
			/* .flags = */ (enum fuse_buf_flags) 0,	\
			/* .mem =   */ NULL,			\
			/* .fd =    */ -1,			\
			/* .pos =   */ 0,			\
		} }						\
	} )
```

https://github.com/libfuse/libfuse/blob/a6a01c4a76c4c8265f7d1f0a0a9b5340b15384ab/include/fuse_common.h#L770